mlflow/diviner.py

```diff
@@ -0,0 +1,452 @@
+import logging
```

Comment:
Can we add a brief module-level docstring that contains a link to the full docs on the MLflow website? Users can see this when they run `help(mlflow.diviner)`, and it might help them get started.

Reply:
added!